pylint_odoo/checkers/no_modules.py

```diff
@@ -357,3 +362,17 @@ def get_decorators_names(self, decorators):
             nodes = decorators.nodes
         return [getattr(decorator, 'attrname', '')
                 for decorator in nodes if decorator is not None]
+
+    @utils.check_messages('translation-errors')
+    def visit_raise(self, node):
+        node_child = node.last_child()
+        if '_(' not in node.last_child().as_string():
+            args = hasattr(node_child, 'keywords') and \
+                node_child.keywords and node_child.args and \
+                (node_child.args + node_child.keywords) or \
+                hasattr(node_child, 'keywords') and node_child.keywords or \
+                node_child.args
+            for argument in args:
+                if argument.as_string()[0] in ['"', '\'']:
```

Comment:
Could you avoid `as_string` validating the type of node to string?
Review follow [check](https://github.com/OCA/pylint-odoo/blob/master/pylint_odoo/checkers/no_modules.py#L219-L230)